iset.mm

```diff
@@ -62258,6 +62290,30 @@ the first case of his notation (simple exponentiation) and subscript it
       FGVOXACFVEVOXBBFABCVFVGVHVIBFVJNVPAFGZVRFGVOVNWIXCWJABFCVKVLAFVJNVM $.
   $}
 
+  $( Any set dominates the empty set.  (Contributed by NM, 26-Oct-2003.)
+     (Revised by Mario Carneiro, 26-Apr-2015.) $)
+  0domg $p |- ( A e. V -> (/) ~<_ A ) $=
+    ( wcel c0 wss cdom wbr 0ss ssdomg mpi ) ABCDAEDAFGAHDABIJ $.
+
+  ${
+    $d A x $.
+    $( A set dominated by the empty set is empty.  (Contributed by NM,
+       22-Nov-2004.) $)
+    dom0 $p |- ( A ~<_ (/) <-> A = (/) ) $=
+      ( vx c0 cdom wbr cv wf1 brdomi wf wa f1f f00 sylib simprd adantl exlimddv
+      wceq cvv wcel 0ex domrefg ax-mp breq1 mpbiri impbii ) ACDEZACQZUFACBFZGZU
+      GBACBHUIUGUFUIUHCQZUGUIACUHIUJUGJACUHKAUHLMNOPUGUFCCDEZCRSUKTCRUAUBACCDUC
+      UDUE $.
+  $}
+
+  ${
+    0sdom.1 $e |- A e. _V $.
+    $( Any set dominates the empty set.  (Contributed by NM, 26-Oct-2003.)
+       (Revised by Mario Carneiro, 26-Apr-2015.) $)
+    0dom $p |- (/) ~<_ A $=
```

Comment:
I would move that right after ~0domg (i.e., swap positions of ~0dom and ~dom0).

Reply:
Where feasible I tend to keep the same order as set.mm. It makes it easier to do things like figure out what has been intuitionized and what has not.

Reply:
Of course, I meant that I would swap positions in both ! But I understand that it's a bit of a hassle. We have to find a better way of keeping set.mm and iset.mm in sync.